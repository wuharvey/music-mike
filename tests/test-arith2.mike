Printfloat (2.22);
Printfloat (3.+.2.5);
Printfloat (10.-.2.);
Printfloat (8.*.4.);
Printfloat (7./.2.);


