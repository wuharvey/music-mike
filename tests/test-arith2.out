2.220000
5.500000
8.000000
32.000000
3.500000
